lib/matplotlib/widgets.py

```diff
@@ -513,25 +509,22 @@ class CheckButtons(AxesWidget):
     For the check buttons to remain responsive you must keep a
     reference to this object.
 
-    The following attributes are exposed
-
-     *ax*
-        The :class:`matplotlib.axes.Axes` instance the buttons are
-        located in
-
-     *labels*
-        List of :class:`matplotlib.text.Text` instances
-
-     *lines*
-        List of (line1, line2) tuples for the x's in the check boxes.
-        These lines exist for each box, but have ``set_visible(False)``
-        when its box is not checked.
-
-     *rectangles*
-        List of :class:`matplotlib.patches.Rectangle` instances
-
-    Connect to the CheckButtons with the :meth:`on_clicked` method
+    Attributes
+    ----------
+    ax : `~matplotlib.axes.Axes`
+        The parent axes for the widget.
+    labels : list of `.Text`
+    rectangles : list of `.Rectangle`
```

Comment:
I find it a little more readable to leave an empty line when there is no parameter description because every parameter stands out with whitespace above and below. But not insisting on this.

Reply:
sure, done